### PR TITLE
style(frontend): change style of Menu button to use ButtonHero

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

New style for `Menu` button using `ButtonHero` component.

Screenshot 2024-07-23 at 19 38 23
Screenshot 2024-07-23 at 19 38 30

### CODE REVIEW

File: src/frontend/src/lib/components/core/Menu.svelte [hunk 1]
```diff
@@ -39,15 +39,9 @@
 	};
 </script>
 
-<button
-	class="user icon desktop-wide"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/core/Menu.svelte:43]: Can we clean-up the style "user"?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Menu.svelte:43]: sure! i would have done in another PR, do you prefer in this one?

Comment (peterpeterparker) [reply src/frontend/src/lib/components/core/Menu.svelte:43]: Another PR is fine, thx

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/core/Menu.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/core/Menu.svelte:43]: Done with PR #1846

**Pull request merged**